k8s/jobs/kustomize/jobs/overlays/common/aries-framework/jsonld/add_contexts.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+apk --no-cache add curl
+
+ENDPOINTS=(adapter-issuer adapter-rp demo-issuer demo-rp governance-vcs holder-vcs issuer-vcs verifier-vcs);
```

Comment:
change this to:
`ENDPOINTS="adapter-issuer adapter-rp demo-issuer demo-rp governance-vcs holder-vcs issuer-vcs verifier-vcs"`